melos.yaml

```diff
@@ -27,7 +27,7 @@ scripts:
         "flutter analyze --fatal-infos ."
       if markdownlint -V >/dev/null 2>&1; then
         printf "\n\033[1mLinting Markdown files ...\033[0;31m\n\n"
-        markdownlint './**/*.md' --ignore './**/CHANGELOG.md' --ignore './**/ios/**' --ignore './**/android/**' \
+        markdownlint -p '.meta/.markdownlintignore' --config '.meta/.mdlrc' './**/*.md' \
```

Comment:
I think it's always nicer to use full option names in config files/scripts so they are easier to read, i.e. here `--ignore-path` instead of `-p`.